storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
I think it makes sense to expose this particular property in `Client.create_bucket`:  it affects all subsequent API requests for the bucket.